src/components/Map/index.js

```diff
@@ -55,10 +55,4 @@ class Map extends React.Component {
   }
 }
 
-function mapStateToProps(state) {
-  return {
-    config: state.config,
-  }
-}
-
-export default connect(mapStateToProps)(Map)
+export default Map
```

Comment:
Is that just a preference change? Or does it do something different if we export default at line 12? Just curious.